pkg/cmd/metrics.go

```diff
@@ -12,38 +13,57 @@ import (
 )
 
 // TODO(saswatamcode): Add flags for URL query params.
-func NewMetricsGetCmd(ctx context.Context) *cobra.Command {
+func NewMetricsGetCmd(ctx context.Context, path ...string) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "get",
 		Short: "Read series, labels & rules (JSON/YAML) of a tenant.",
 		Long:  "Read series, labels & rules (JSON/YAML) of a tenant.",
 	}
 
+	var seriesMatchers []string
 	seriesCmd := &cobra.Command{
 		Use:   "series",
 		Short: "Get series of a tenant.",
 		Long:  "Get series of a tenant.",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			b, err := config.DoMetricsGetReq(ctx, logger, "/api/v1/series")
+			u := "/api/v1/series"
+			params := url.Values{}
+
+			for _, s := range seriesMatchers {
+				params.Add("match[]", s)
+			}
+
+			if len(params.Encode()) != 0 {
+				u = u + "?" + params.Encode()
+			}
+
+			b, err := config.DoMetricsGetReq(ctx, logger, u, path...)
 			if err != nil {
 				return err
 			}
 
-			return prettyPrintJSON(b)
+			fmt.Fprintln(cmd.OutOrStdout(), "this should work")
```

Comment:
nit. looks like we don't need this line?

Reply:
Ah sorry, missed this line, let me remove!